assignments/array-methods.js

```diff
@@ -54,30 +54,93 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running
+// bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(function(newNames) {
+  let bothNames = `${newNames.first_name} ${newNames.last_name}`;
+  fullName.push(bothNames)
+});
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
-// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
+// The event director needs to have all the runner's first names converted to uppercase
+// because the director BECAME DRUNK WITH POWER. Convert each first name into all caps
+// and log the result
 let allCaps = [];
-console.log(allCaps); 
+
+allCaps = runners.map(function(newNames) {
+  let firstNames = newNames.first_name.toUpperCase();
+  return firstNames
+});
+
```

Comment:
const allCaps = runners.map(runner => runner.first_name.toUpperCase());
//showing you how it's written as an arrow function.